src/PageTypes/BlocksPage.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace SilverStripe\StartupThemeComponents\PageTypes;
+
+use Page;
+use SilverStripe\Forms\CheckboxField;
+
+class BlocksPage extends Page
+{
```

Comment:
We should have a line after the opening brace and before the closing brace

Reply:
Will do!